addons/l10n_ae/__manifest__.py

```diff
@@ -1,31 +1,29 @@
 # -*- coding: utf-8 -*-
 # Part of Odoo. See LICENSE file for full copyright and licensing details.
-
-# Copyright (C) 2014 Tech Receptives (<http://techreceptives.com>)
-
 {
     'name': 'U.A.E. - Accounting',
-    'author': 'Tech Receptives',
+    'author': 'sal-odoo',
```

Comment:
We never really change to ourselves.  (only for external)  But does this come from Tech Receptives originally?  (as they are India based)

Reply:
Yeah it originally comes form the partner Tech Receptives. Sure, I'll remove my name. Should I keep the partner's name?

Reply:
Sure

Reply:
FIxed